css/style.css

```diff
@@ -57,7 +57,7 @@ video {
 	left: 20px;
 }
 
-.main-nav__logo__image {
+.main-nav__logo-image {
 	width: 89px;
```

Comment:
this width should be set in `main-nav__logo`, `main-nav__logo-image` should not exist

Reply:
But the image has a container `main-nav__logo`.

Reply:
@camigomez35 exactly, therefore `width: 89px;` should be on that container, not the image.
images should always be `width: 100%;` which is set on the global styles.

Reply:
This comes from an old PR...